simplq/src/store/appSlice.js

```diff
@@ -20,28 +24,30 @@ const appSlice = createSlice({
       state.notificationPermission = action.payload;
     },
   },
-  extraReducers: {
-    [deleteQueue.pending]: (state) => {
-      state.infoText = `Deleting queue...`;
-    },
-    [deleteQueue.rejected]: (state, action) => {
-      state.errorText = action.error.message;
-    },
-    [deleteQueue.fulfilled]: (state, action) => {
-      state.infoText = `Deleted ${action.payload.queueName}`;
-    },
-    [getQueueStatusByName.rejected]: (state, action) => {
-      state.errorText = action.payload.message;
-    },
-    [joinQueue.pending]: (state) => {
-      state.infoText = `Adding to queue...`;
-    },
-    [joinQueue.rejected]: (state, action) => {
-      state.errorText = action.error.message;
-    },
-    [joinQueue.fulfilled]: (state, action) => {
-      state.infoText = `Added to ${action.payload.queueName}`;
-    },
+  extraReducers: (builder) => {
+    builder
+      .addCase(joinQueue.pending, (state) => {
+        state.infoText = `Adding to queue...`;
```

Comment:
What's the reason for using builder.addCase?? Is it somehow better than the old version?

Reply:
Yes, it's the recommended way, plus you can do advanced matching, see the last case in the chain.

https://redux-toolkit.js.org/api/createSlice#the-extrareducers-builder-callback-notation

Reply:
Okay, got it :+1: